puppet/modules/fastly_purge/manifests/init.pp

```diff
@@ -0,0 +1,20 @@
+# @summary scripts to manage fastly CDN purging
+#
+class fastly_purge(
+) {
+  file { '/usr/local/bin/fastly-purge':
+    ensure  => file,
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0755',
+    content => file('web/fastly-purge.sh'),
```

Comment:
CI is unhappy about this and the other file. You can make it dynamic:
```suggestion
    content => file("${module_name}/fastly-purge.sh"),
```

Or hardcoded:
```suggestion
    content => file('fastly_purge/fastly-purge.sh'),
```

Reply:
right, that's what you get when looking at "wth did I break between meetings" :) (fixed)